lib/cog/bootstrap.ex

```diff
@@ -54,6 +67,65 @@ defmodule Cog.Bootstrap do
     end)
   end
 
+  # Bootstraps using environment variables if they are present.
+  @doc """
+  Create an administrative user using values from the environment, if present.
+  If any of the following variables are not set, the bootstrap will not be
+  performed. Required variables:
+
+  COG_BOOTSTRAP_USERNAME
+  COG_BOOTSTRAP_PASSWORD
+  COG_BOOTSTRAP_EMAIL_ADDRESS
+  COG_BOOTSTRAP_FIRST_NAME
+  COG_BOOTSTRAP_LAST_NAME
+
+  Additionally, if the user bootstrap completes successfully and the RELAY_ID
+  and RELAY_COG_TOKEN variables are set, a relay will be created using the
+  variables and assigned to a default relay group, which will also be created.
+  If the relevant variables are not present, this step will be skipped.
+
+  If any creation attempt fails, the entire bootstrap process will be rolled
+  back.
+  """
+  def maybe_bootstrap do
+    unless is_bootstrapped? do
+      Repo.transaction fn ->
+        if bootstrap_from_env, do: relay_from_env(System.get_env)
+      end
+
+      true
+    end
+
+    false
```

Comment:
I ... don't remember. Looking at the code now it doesn't seem to be used. I think I just wanted to indicate whether a bootstrap actually  happened or not as a result of that call. I can easily take taht out if we want.